storage/map_files_downloader.hpp

```diff
@@ -87,5 +95,8 @@ class MapFilesDownloader
   ServersList m_serversList;
   bool m_isServersListRequested = false;
   DownloadingPolicy * m_downloadingPolicy = nullptr;
+
+  /// Queue which is used while servers list receiving.
```

Comment:
// This queue accumulates download requests before the servers list is received
// on the network thread to reroute them later to the UI thread.
// The rerouting cannot be done directly because of the possibility of requests being cancelled.

Reply:
поддерживаю, но только считаю, что часть 
```
to reroute them later to the UI thread.
// The rerouting cannot be done directly because of the possibility of requests being cancelled.
```
лишняя

Reply:
сейчас запросы изначально добавляются в очередь на главном потоке и используются потом на главном потоке. Никакого рероутинга нету.

Reply:
поправил коммент